src/helper/UserSession.cpp

```diff
@@ -353,11 +353,20 @@ namespace SDDM {
             //we want to redirect after we setuid so that the log file is owned by the user
 
             // determine stderr log file based on session type
-            QString sessionLog = QStringLiteral("%1/%2")
-                    .arg(QString::fromLocal8Bit(pw.pw_dir))
-                    .arg(sessionType == QLatin1String("x11")
-                         ? mainConfig.X11.SessionLogFile.get()
-                         : mainConfig.Wayland.SessionLogFile.get());
+            QString sessionLog;
+            if ( (sessionType == QLatin1String("x11")) ||
```

Comment:
You can invert this and check for `wayland` instead

Reply:
Done